### ISSUE TITLE
Initialised var not later being mangled

### ISSUE THREAD

**Issue by adoxa** [putoutjs/minify#14]
- **Minify Version**: v11.2.0
- **Node Version**: v20.13.1
- **OS**: Windows 10 64-bit

This sequence:

```javascript
!function() {
    let string_var = "1";
    string_var += "2";
    console.log(string_var);
}()
```

is incorrectly minified to this:

```javascript
!function(){string_var+='2';console.log('1')}();
```

Assigning it separately does work:

```javascript
!function() {
    let string_var;
    string_var = "1";
    string_var += "2";
    console.log(string_var);
}()
```

becomes:

```javascript
!function(){let a;a='1';a+='2';console.log(a)}();
```

**Comment by coderaiser** [comment]
Thanks you! Just fixed, and landed in [v3.13.0](https://github.com/putoutjs/minify/releases/tag/v3.13.0) 🎉.

Is it works for you?

**Comment by adoxa** [comment]
Yep, that fixed it.
